main.py

```diff
@@ -220,9 +220,13 @@ def add_result(
                     + process_result.stderr
                 )
 
+            probable_result = TestResult.PROCESS_ERROR
+            if process_result.returncode == -signal.SIGALRM:
```

Comment:
is `returncode` actually negative? :thinking:

Reply:
It is when killed by signal:

> returncode
>     Exit status of the child process. If the process exited due to a signal, this will be the negative signal number.

From: https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError.returncode

Reply:
Ah, TIL!